CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [7.0.0] - 2021-04-06
+
+### Added
+
+- Ability to indicate if a transaction was added from the users local device and account creation time ([#436](https://github.com/MetaMask/controllers/pull/436))
+
+### Changed
+
+- **BREAKING:** Organize assets by chainid ([#435](https://github.com/MetaMask/controllers/pull/435))
+- Support longer token symbols via wallet_watchAsset ([#433](https://github.com/MetaMask/controllers/pull/433))
+- Add comment to explain why camelcase is disabled ([#432](https://github.com/MetaMask/controllers/pull/432))
```

Comment:
Nit: Not super important, but generally non-functional changes like comments can be omitted.

It's a judgement call, because sometimes it's worth calling attention to changes even when they're non-functional (e.g. "improved test coverage" might be of interest to consumers). But adding a comment is pretty far from being notable I think!

```suggestion
```

Reply:
👌   sounds good